### ISSUE TITLE
DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead

### ISSUE THREAD

**Issue by erickwilder** [heyscrumpy/tiptap#38]
I've just cloned the repo, installed all dependencies and ran the examples and got this:

```
yarn run build:examples                                                                                                                                                                                                                  
$ ./node_modules/@babel/node/bin/babel-node.js build/examples/build.js --env=production
⠋ Building …(node:27582) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
```

What about making `webpack@4` the default for development purposes? I don't think it will break anything for _users_ of the library

**Comment by philippkuehn** [comment]
Hey @erickwilder, this deprecation warning is not related to tiptap. It's caused by any webpack plugin or loader for the local environment. The local setup is already based on webpack 4.

**Comment by vittal288** [comment]
I have upgraded from angular 5 to angular 7 , post this I am trying to run the application using ng serve command, I am getting this error and any solution is appreciated.

**Comment by eyal-elkevity** [comment]
I had this issue when doing upgrade from ng6 to 8. but the main issue was that I was ejected and had my own build in webpack, and decided to rollback to ng cli. the warning appeared because I had my own old version of webpack libraries, which were no longer needed. so I removed them, but then the serve/build failed because angular-devkit needed webpack libs. and then I realised, that angular-devkit were using my old webpack libs and this caused the warning. so I reinstalled angular-devkit (install to specific version with `--force`) and BAM - everything worked out fine and the warning was gone.
